test/application_system_test_case.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 require 'capybara/poltergeist'
 
-DIMENSIONS = [1400, 1400].freeze
+DIMENSIONS = [1200, 3000].freeze
```

Comment:
because test will fail with poltergeist but not with chrome... that simple. Fix Poltergeist, and I'll change them back